allensdk/core/mouse_connectivity_cache.py

```diff
@@ -513,6 +513,64 @@ def get_experiment_structure_unionizes(self, experiment_id,
                                                 writer=lambda p, x : pd.DataFrame(x).to_csv(p),
                                                 reader=pd.DataFrame.from_csv)
 
+    def rank_structures(self, experiment_ids, is_injection, structure_ids, hemisphere_ids=None, 
+                        rank_on='normalized_projection_volume', n=5, threshold=10**-2):
+        '''Produces one or more (per experiment) ranked lists of brain structures, using a specified
+
+        Parameters
+        ----------
+        experiment_ids : list of int
+            Obtain injection_structures for these experiments.
+        is_injection : boolean
+            Use data from only injection (or non-injection) unionizes
+        structure_ids : list of int
+            Consider only these structures. It is a good idea to make sure that these structures are not spatially 
+            overlapping; otherwise your results will contain redundant information.
+        hemisphere_ids : list of int, optional
+            Consider only these hemispheres (1: left, 2: right, 3: both). Like with structures, 
+            you might get redundant results if you select overlapping options.
+        rank_on : str, optional
+            Rank unionize data using this field (descending). Defaults to normalized_projection_volume.
+        n : int, optional
+            Return only the top n structures
+        threshold : float, optional
```

Comment:
can this be `normalized_projection_volume_threshold` or `npv_threshold`?

Reply:
(unless you decide to generalize this)

Reply:
Can this default to summary structures?

Reply:
I like that idea -- `structure_ids` to default to the summary structure set.